src/path_support.cc

```diff
@@ -28,7 +28,7 @@ namespace grpc_labview
         {
             char path[MAX_PATH];
             if (GetModuleFileNameA(hModule, path, MAX_PATH)) {
```

Comment:
LabVIEW 2025 Q3 will be long-path aware: https://forums.ni.com/t5/LabVIEW-Public-Beta-Program-in/New-Feature-Support-for-Windows-Long-Paths/td-p/4434839

This code is using a fixed-size path buffer of size `MAX_PATH`, so it will truncate the path to 260 characters.

Consider making this code long-path aware:
```suggestion
            std::string path(MAX_PATH);
            DWORD result = GetModuleFileNameA(hModule, path.data(), path.size());
            while (result >= path.size() && GetLastError() == ERROR_INSUFFICIENT_BUFFER) {
                path.resize(path.size() * 2);
                result = GetModuleFileNameA(hModule, path.data(), path.size());
            }
            path.resize(result); // remove padding at end
            if (result) {
```

Reply:
I don't think that's necessary until LabVIEW supports passing a Unicode string to the call library function node. Until then, the path to the DLL must be representable in the current ANSI code page.